subprojects/packagefiles/openssl/meson.build

```
@@ -357,40 +180,32 @@ if get_option('default_library') == 'static'
   ]
 endif
 
-foreach define : defines + openssl_defines
-  c_args += ['-D' + define]
+foreach def : defines
+  c_args += '-D' + def
```

`c_args` is an array here:
```suggestion
  c_args += ['-D' + def]
```

---

Meson language is special here, `c_args += '-D' + def` does add string to array without having to created a temporary 1-item array.